src/TLuaInterpreterMapper.cpp

```diff
@@ -2474,6 +2481,7 @@ int TLuaInterpreter::lockExit(lua_State* L)
     if (pR) {
         pR->setExitLock(dir, b);
         host.mpMap->setUnsaved(__func__);
+        host.mpMap->update();
```

Comment:
Locking an exit does not produce any visible changes - so this is not needed.

Reply:
It might in the future - and if someone isn't being super careful in noticing that the map update call is missing, we'll have an issue.

Refreshing the map just once is pretty quick - better not to leave footguns around.